app/lib/actions/candlepin/listen_on_candlepin_events.rb

```diff
@@ -19,7 +19,7 @@ def notify_connected
       end
 
       def notify_not_connected(message)
-        @suspended_action << Actions::Candlepin::ListenOnCandlepinEvents::NotConnected[message]
+        @suspended_action << Actions::Candlepin::ListenOnCandlepinEvents::ReconnectOnSleep[message]
```

Comment:
You can `world.clock.ping suspended_action, 5, NotConnected` directly here: the `ReconnectOnSleep` to reduce the amount of events we pass (it's quite hard to read), Also `Reconnect` instead of `NotConnected` might be a better name for the event then